app/views/works/_curator_controlled_display.html.erb

```diff
@@ -1,13 +1,13 @@
 <div class="section-title">
   <!-- DOI field (mostly for legacy DSpace works) -->
   <div class="field">
-    DOI <span class="text-muted field-hint">(enter the Digital Object Identifier, e.g. 10.80021/62qy-nj36)</span><br>
```

Comment:
Any reason to remove the hints for the DOI and the ARK?

Reply:
Yes, since this is only a display form, I thought the hints were not desired.  Aka there is nothing to enter in this form.  It is extraneous to this PR, I just noticed it when I was on the page.  I can put it in it's won PR if desired.

Reply:
ah! that makes sense. I actually had done just the same (display on edit, not on show) on the branch that I am working :D -- keep it in your PR and I'll adjust mine when I rebase.